services/src/northbound/src/main/java/org/openkilda/northbound/service/impl/LinkServiceImpl.java

```diff
@@ -158,14 +162,35 @@
                 .thenApply(responses -> getLinkPropsResult(responses, errors));
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public CompletableFuture<LinkMaxBandwidthDto> updateLinkBandwidth(SwitchId srcSwitch, Integer srcPort,
+                                                                      SwitchId dstSwitch, Integer dstPort,
+                                                                      LinkMaxBandwidthRequest input) {
+        org.openkilda.messaging.model.LinkPropsDto linkProps = org.openkilda.messaging.model.LinkPropsDto.builder()
+                .source(new NetworkEndpoint(srcSwitch, srcPort))
+                .dest(new NetworkEndpoint(dstSwitch, dstPort))
+                .props(ImmutableMap.of(LinkProps.MAX_BANDWIDTH_PROP_NAME,
+                        String.valueOf(input.getMaximumBandwidth())))
```

Comment:
what's the purpose of `String.valueOf` here?

Reply:
Map in link props is <String, String>